.buildkite/scripts/publish.sh

```diff
@@ -11,7 +11,7 @@ aws s3 sync --delete \
 
 aws s3 sync --delete \
     "storybook/public" \
-    "s3://${KAIZEN_DOMAIN_NAME}${KAIZEN_BASE_PATH}/storybook-static"
+    "s3://${KAIZEN_DOMAIN_NAME}${KAIZEN_BASE_PATH}/storybook"
```

Comment:
I _think_ this will automagically change the route from `http://cultureamp.design/storybook-static` to `http://cultureamp.design/storybook` - is this correct @zioroboco ?

Reply:
Yep. This will write the output of the storybook build to the new path in S3, including for dev deploys.

The s3/cloudfront stack takes urls like `cultureamp.com/some/path` and return whatever it finds at `s3://<branch-routing-bits>/some/path/index.html`. The line you've changed will deploy assets to the `/storybook/*` path.

Reply:
Which, by the way, means if you ever wanted to deploy anything else (maybe an example page, like we were discussing), you only need to build it in CI and deploy it to whichever path you want, as above.

This will also work for branch deploys.

Reply:
Neat, this is good to know.